k8s/operator/manifests/operator.yaml

```diff
@@ -87,7 +87,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: keramik-operator
-  namespace: default
+  namespace: keramik
```

Comment:
I believe this was intentionally default. IIUC, the operator doesn't itself run in the `keramik` namespace, though @nathanielc might have a better idea if it matters much.